visualization/app/codeCharta/e2e/url.e2e.ts

```diff
@@ -56,7 +56,6 @@ describe("codecharta", () => {
 	it("should load data when file parameters in url are valid", async () => {
 		await mockResponses()
 		await goto(`${CC_URL}?file=fileOne.json&file=fileTwo.json`)
-		await checkSelectedFileName("Sample Project with Edges")
```

Comment:
Ideally there would be a test to check the behavior if the mode parameter is present and another one if not.